akka-docs/src/test/java/jdocs/stream/operators/SinkDocExamples.java

```diff
@@ -139,6 +141,30 @@ static void ignoreExample() {
     // #ignore
   }
 
+  static void asPublisherExample() {
+    // #asPublisher
+    Source<Integer, NotUsed> source = Source.range(1, 5);
+
+    Publisher<Integer> publisherTrue = source.runWith(Sink.asPublisher(AsPublisher.WITH_FANOUT), system);
+    CompletionStage<Integer> resultFromFirstSubscriberTrue =
+            Source.fromPublisher(publisherTrue).runWith(Sink.fold(0, (acc, element) -> acc + element), system);
```

Comment:
I think a `Sink.foreach(println)` would be more clear than an aggregate result in showing that each element comes to the downstream.